src/features/ticket/api/ticketOption.ts

```
@@ -0,0 +1,68 @@
+// 티켓에 부착된 티켓 옵션 부착 취소
+export const detachTicketOption = async (
+  ticketId: number,
+  ticketOptionId: number
+): Promise<ApiResponse<{}>> => {
+  const response = await axiosClient.delete<ApiResponse<{}>>(`/ticket-option-assignments`, {
+    params: {
+      ticketId,
+      ticketOptionId,
+    },
+  });
+  return response.data;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)